_posts/_2021-05-14-opensearch-beta-announcement.markdown

```diff
@@ -8,68 +8,56 @@ date:   2021-05-14 01:01:01 -0700
-- Reporting Dashboards Plugin ([Release Notes](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-beta1.md
-))
-- Reporting Opensearch Plugin ([Release Notes](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-beta1.md
-))
-- Trace Analytics Dashboards Plugin ([Release Notes](https://github.com/opensearch-project/trace-analytics/blob/main/release-notes/opensearch-trace-analytics.release-notes-1.0.0.0-beta1.md))
-- Gantt Chards - Dashboards Visualization plugin ([Release Notes](https://github.com/opensearch-project/dashboards-visualizations/blob/main/release-notes/opensearch-dashboards-visualizations.release-notes-1.0.0.0-beta1.md))
-- Async-Search ([Release Notes](https://github.com/opensearch-project/asynchronous-search/blob/main/release-notes/opensearch-asynchronous-search.release-notes-1.0.0.0-beta1.md))
-- OpenSearch CLI ([Release Notes](https://github.com/opensearch-project/opensearch-cli/blob/main/release-notes/opensearch-cli.release-notes.1.0.0-beta1.md))
-- Common-Utils ([Release Notes](https://github.com/opensearch-project/common-utils/blob/main/release-notes/opensearch-common-utils.release-notes-1.0.0.0-beta1.md))
-- Data Prepper (Release Notes)
-
-## What's Next?
-
-**TODO** For next steps... 
+We are excited to release the OpenSearch Beta 1.0 (derived from Elasticsearch 7.10.2) and OpenSearch Dashboards Beta 1.0 (derived from Kibana 7.10.2). With this beta release, we  have refactored all the Open Distro for Elasticsearch plugins to work with OpenSearch and provide the community with downloadable artifacts (Linux tars and Docker images) to run OpenSearch and OpenSearch Dashboards with these plugins installed. Features released with this beta inlcude;
+
+- Advanced Security
+- SQL Query Syntax
+- Anomaly Detection
+- Alerting
+- Piped Processing Language
+
+...and more ([OpenSearch](https://github.com/opensearch-project/OpenSearch) and [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards)). 
+
+We would like to get your feedback from this beta release - please let us know if there are any features you would like or bugs you identify. You can submit pull requests, write documentation, open issues (either on [OpenSearch](https://github.com/opensearch-project/OpenSearch/issues) or [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/issues)), or simply read the [news](https://opensearch.org/blog/). If you would like to install or run this project please see the Developer Guide ([OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md) and [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md)) to get started.
+
+#### Plugins Included in Beta are:
```

Comment:
let's just make the name of the plugin the link to the repo too.

Reply:
Makes sense - I'll update this next.

Reply:
Done!